com.ibm.wala.scandroid/source/org/scandroid/prefixtransfer/UriPrefixTransferGraph.java

```diff
@@ -48,15 +48,8 @@
 
 package org.scandroid.prefixtransfer;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Star import

Reply:
I agree. My IDE was trying to be helpful behind my back. I have reconfigured my IDE and will correct this in a forthcoming group of updates for this pull request.